api/db-cdr/generate-cdr/import-gcs-data.sh

```diff
@@ -56,7 +56,18 @@ function mysqlimport_table () {
    file=$2
 
    echo "Mysql importing $file into $db.$table..."
-   mysqlimport --ignore-lines=1 --fields-terminated-by=, --fields-enclosed-by='"' \
+   mysqlimport --ignore-lines=1 --fields-terminated-by=, --fields-optionally-enclosed-by='"' \
+      --verbose --local -h ${DB_HOST} --port ${DB_PORT} \
+      -u root -p${MYSQL_ROOT_PASSWORD} $db $file
+}
+
+# Function for mysqlimport concept_synonym
+function mysqlimport_concept_synonym_table () {
+   db=$1
+   file=$2
+
+   echo "Mysql importing $file into $db.$table..."
+   mysqlimport --ignore-lines=1 --fields-terminated-by=, --fields-optionally-enclosed-by='"' --columns=concept_id,concept_synonym_name,language_concept_id \
```

Comment:
Why do we need this ? Wouldn't we just want to make the csv have the right columns? When i ran the command on the csv it did just fine on my computer. 

Also, Aare we sure we don't want to just add a dummy auto inc  column like we do for the other tables ?  We only did that for jpa to be happy with relationships and such.  But like achilles_results and any table that doesn't have a single pk we just made one up.  Maybe better to just have the multiple pk if we need it for relationships. 

Anway, just thoughts on that for future reference.

Reply:
The import did not work fine when i tried without this.
So, added the column names.